config/core/test-configs.yaml

```diff
@@ -959,6 +959,12 @@ device_types:
       - blocklist: *device_config_filter
       - blocklist: {defconfig: ['multi_v7_defconfig+kselftest']}
 
+  juno-r0-uboot:
```

Comment:
The LAVA device type in upstream LAVA is [`juno-uboot`](https://git.lavasoftware.org/lava/lava/-/blob/master/etc/dispatcher-config/device-types/juno-uboot.jinja2) so you either need to add `name: juno-uboot` to the device type definition here or send a `juno-r0-uboot` variant upstream.

Reply:
The upstream LAVA device type appears to be both broken and to look like it's got stuff to reflash the firmware in there which I'm actively trying to avoid. The whole point with this type as with the -acpi version is that there is upstream support for these devices but it pulls in flashing the firmware which both complicates generating the job definitions and slows everything down.

It's not clear why we have this requirement for some devices but not others - there wasn't an issue with the ACPI version of this?

Reply:
Ah, no - the upstream LAVA type does seem to be able to avoid reflashing. Doesn't really help with the bugginess though (or the fact that it's independent of revision of the board which means that there is no way to figure out the correct DT).

Reply:
Never mind, looks like upstream uses no revision as a synonym for r0... will switch to that.

Reply:
> It's not clear why we have this requirement for some devices but not others - there wasn't an issue with the ACPI version of this?

It was an oversight on my part in the previous PR with ACPI, I just assumed that Juno device types were all good in upstream LAVA.

Reply:
> Never mind, looks like upstream uses no revision as a synonym for r0... will switch to that.

OK thanks, that should simplify things a bit. I just wonder how we might cope with different Juno revisions if we have some online at some point.